src/python/WMCore/Cache/WMConfigCache.py

```diff
@@ -432,7 +437,7 @@ def addConfig(self, newConfig, psetHash=None):
 
         """
         # The newConfig parameter is a URL suitable for passing to urlopen.
-        with closing(urllib.urlopen(newConfig)) as f:
+        with closing(urllib.request.urlopen(newConfig)) as f:
```

Comment:
As the commit message states:

>   ACHTUNG! This may require changes to code that use
  `WMCore.Cache.WMConfigCache.ConfigCache.addConfig`

For example [cms-PdmV/wmcontrol/modules/wma.py](https://github.com/cms-PdmV/wmcontrol/blob/1cd4db611d167f73ed4f9ae109ed360a3392eaae/modules/wma.py#L272)

When merging this, we must state that if someone depends on __some specific exception__ that this function may raise in case of error, then they need to update their code. If they do not care about exceptions, then they should be fine.